scripts/cfos/tests/README.md

```diff
@@ -0,0 +1,24 @@
+## Test schema file setup ##
+You can run the code using the test schema by adding the parameter "-t True" to run in testing mode (examples shown below in How to Run section)
+
+## How to Run ##
+### test fail ### 
+python3 make_dataset_data_tables.py -d schema1 -x tests/CFoS_Template_Test_Schema1_Fail.xlsx -p broad-cfos-data-platform1 -w cFOS_automation_testing -t True
+
+### test pass ###
+python3 make_dataset_data_tables.py -d schema1 -x tests/CFoS_Template_Test_Schema1_Pass.xlsx -p broad-cfos-data-platform1 -w cFOS_automation_testing -t True
+
+
+## Validating outcomes ##
```

Comment:
I would move this entire section to the top-level README - I think this is useful for the user to see. I'd also love to see the literal contents of `validation_errors.csv` when run on the failing test, but if it's similar enough to the examples you gave below, I think that's ok.

Reply:
The examples are excerpts of the `validation_errors.csv`. I think showing the whole file would be overwhelming unnecessarily unless someone is actively trying to debug their data.

Reply:
Should I just get rid of the tests README then? ANd move it all to the main one?